src/components/BoardOptionsMenu/ExportModal/ExportModal.tsx

```diff
@@ -16,15 +16,15 @@ const ExportModal = ({ closeExportModal, handleExportList }: ExportModalProps) =
       className="z-50 min-h-full min-w-full flex fixed top-0 left-0 justify-center items-center bg-black bg-opacity-30"
     >
       <div className="relative px-4 w-full max-w-sm h-full md:h-auto mb-36">
-        <div className="relative bg-trello-gray-100 rounded-lg shadow dark:bg-gray-700">
+        <div className="relative bg-trello-gray-100 rounded-lg shadow">
           <div className="flex p-6">
-            <div className="">
+            <div>
               <select
                 value={selectedListId}
                 onChange={handleChange}
                 className="form-select p-6 appearance-none w-md px-3 py-1.5 text-base font-normal text-gray-700 bg-clip-padding bg-no-repeat border border-solid border-gray-300 rounded transition ease-in-out m-0 focus:text-gray-700 focus:bg-white focus:border-blue-600 focus:outline-none"
               >
-                <option selected value="" disabled>
+                <option value="" disabled>
```

Comment:
`selected` shouldn't be used here as it'll log an error in the tests. This is because React uses the value attribute on the select tag to determine what is selected, rather than this selected attribute on the option